_readme/README.jl

```diff
@@ -132,7 +132,7 @@ save("images/3d_Fig3.png", fig)
 
 # ### Animations and interactivity
 
-# Finally, by using Makie [Observables](https://makie.juliaplots.org/stable/interaction/nodes.html), we
+# Finally, by using Makie [Observables](http://makie.juliaplots.org/v0.15.1/documentation/nodes/index.html), we
```

Comment:
Do you actually want a hardcoded version link, or a version-independent link better?

Reply:
I guess the first one is preferrable, right?

Reply:
Not sure which is the best option. Ideally it should be an autogenerated link. Perhaps you can extract the current version of Makie.jl from the README.jl using package manager functions and then auto-generate the link. (I have never done something like that, but does not seem  hard).

Reply:
https://www.juliabloggers.com/how-to-check-the-version-of-a-package/

Reply:
I can get the version of Makie as a string but not convert it to Markdown. See:
```julia
using Pkg, Makie
Makie_version = Pkg.TOML.parsefile(joinpath(pkgdir(Makie), "Project.toml"))["version"] # which gives "0.15.1"
```
And how to convert Makie_version to md text?

Reply:
I would generate the md file with a TAG e.g. 

```
# Finally, by using Makie [Observables](http://makie.juliaplots.org/XXXMAKIEVERSIONXXX/documentation/nodes/index.html)
```
and then do a source to source replace after generating it

Reply:
https://discourse.julialang.org/t/best-practice-for-streaming-a-template-file-with-replaced-strings/6469